ui/components/imageWorkspace/src/components/SaveShape/SaveShapeForm.svelte

```diff
@@ -76,6 +81,21 @@
 
       return [...oldObjects, ...(newObject ? [newObject] : [])];
     });
+
+    // add new inputs to lists of available values
+    if ("objects" in $itemFeaturesAvailableValues) {
```

Comment:
this function looks the same as in `TextInputFeature.svelte`. Maybe you could abstract it in a function in an api file ?